chaoscenter/graphql/server/pkg/database/mongodb/init.go

```diff
@@ -58,8 +58,8 @@ var (
 		ChaosExperimentCollection:     "chaosExperiments",
 		ChaosExperimentRunsCollection: "chaosExperimentRuns",
 		ChaosHubCollection:            "chaosHubs",
-		ImageRegistryCollection:       "image-registry",
-		ServerConfigCollection:        "server-config",
+		ImageRegistryCollection:       "imageRegistry",
```

Comment:
Just a thought: imageRegistry collection can be renamed to chaosImageRegistry and same for gitops.

Reply:
These are not chaos resources that's why I did not rename it.